### ISSUE TITLE
Irish Grid error with valid coordinates

### ISSUE THREAD

**Issue by AlEscarcha** [dvdoug/PHPCoord#62]
Good morning, first of all thank you for this great Lib! Really helpful.

I'm having trouble converting to the Irish Grid (EPSG_TM75_IRISH_GRID) in some valid coordinates (Unable to perform conversion, please file a bug if you think this is incorrect)

```
        $from = GeographicPoint::create(
            new Degree(54.040039),
            new Degree(-6.1846184),
            null,
            Geographic2D::fromSRID(Geographic2D::EPSG_WGS_84),
        );

        $toOSGrid = Projected::fromSRID(Projected::EPSG_TM75_IRISH_GRID);
        $to = $from->convert($toOSGrid);

```

According to https://irish.gridreferencefinder.com/?gr=J1896211603%7C54.04003822493_s__c__s_-6.1846160888672%7C1&t=54.04003822493%20%2C%20-6.1846160888672&v=r this is valid. If the latitude is below 54 it works perfectly.

We're on PHP 7.4, so if any possible fix could be ported to the ^4 branch it'd be amazing!

**Comment by dvdoug** [comment]
The site you linked to gives slightly different results (off by

**Comment by dvdoug** [comment]
Fixed in v4.7.1
